bcolz/ctable.py

```diff
@@ -644,13 +644,17 @@ def copy(self, **kwargs):
         return ccopy
 
     @staticmethod
-    def fromdataframe(df, **kwargs):
+    def fromdataframe(df, index_as_columns=False, **kwargs):
```

Comment:
I had the backward compatibility in mind, but I do agree that `index_as_columns` to `True` as a default makes more sense.